src/components/home/elements/TokenRecord.vue

```diff
@@ -0,0 +1,474 @@
+                this.$http.get(turl).then(response => {
+                    let value = BigNumber(response.body.balance).dividedBy(this.unity)
+                    Vue.set(this.tokenBalances, addr, value)
+                }, respError => {
+                })
+            }
+            for (const addr in this.coldAddresses) {
+                Vue.set(this.tokenBalances, addr, BigNumber(0))
+                let turl = NODE_IP + '/contract/balance/' + addr + '/' + this.tokenId
+                this.$http.get(turl).then(response => {
+                    let value = BigNumber(response.body.balance).dividedBy(this.unity)
+                    Vue.set(this.tokenBalances, addr, value)
+                }, respError => {
+                })
+            }
+        },
+        getTokenInfo() {
+            const tokenUrl = NODE_IP + '/contract/tokenInfo/' + this.tokenId
+            this.$http.get(tokenUrl).then(response => {
+                this.tokens = response.body
+                this.unity = BigNumber(this.tokens.unity)
+            }, respError => {
+            })
+
+            const url = NODE_IP + '/contract/info/' + this.contract
+            this.$http.get(url).then(response => {
+                this.issuer = response.body.info[0].data
+                this.maker = response.body.info[1].data
+                this.contractId = response.body.contractId
+            }, respError => {
+                this.issuer = 'Failed to get issuer'
+                this.registerTime = 'Failed to get time'
+            })
+
+            var contractId = transaction.tokenIDToContractID(this.tokenId)
+            const url2 = NODE_IP + '/contract/content/' + contractId
+            this.$http.get(url2).then(response => {
+                var tokentype = response.body.textual.descriptors
+                this.functionIndex = -9
+                if (tokentype === CONTRACT_DESCRIPTOR) {
+                    this.functionIndex = SEND_FUNCIDX
+                } else if (tokentype === CONTRACT_WITH_SPLIT_DESCRIPTOR) {
+                    this.functionIndex = SEND_FUNCIDX_SPLIT
+                } else {
+                    this.functionIndex = -999
+                }
+            }, respError => {
+                console.log('failed to load tokentype ')
+            })
+            this.updateBalance()
+        },
+        showModal() {
+            this.getTokenInfo()
+            this.$root.$emit('bv::show::modal', 'tokenInfoModal_' + this.tokenId)
+        },
+        sendToken() {
+            this.getTokenBalances()
+            this.getTokenInfo()
+            this.$root.$emit('bv::show::modal', 'sendTokenModal_' + this.tokenId)
+        },
+        issueToken() {
+            this.getTokenInfo()
+            this.$root.$emit('bv::show::modal', 'issueTokenModal_' + this.tokenId)
+        },
+        burnToken() {
+            this.$root.$emit('bv::show::modal', 'burnTokenModal_' + this.tokenId)
```

Comment:
why there is no `this.getTokenInfo()` in burnToken? and what the usage of `this.getTokenInfo()` in `sendToken()`? if only need to update the balance, why not use `this.updateBalance()` directly?

Reply:
Removed useless getting balance in sendToken(). And added token info update in burnToken().